modules/account-lib/package.json

```diff
@@ -34,6 +34,7 @@
     "@bitgo/sdk-coin-bld": "^1.4.11",
     "@bitgo/sdk-coin-bsc": "^3.8.11",
     "@bitgo/sdk-coin-celo": "^1.5.11",
+    "@bitgo/sdk-coin-core": "^1.0.0",
```

Comment:
Do other projects need it in `account-lib`? If not, we try to avoid putting it here.

Reply:
It will be required in Wallet Platform